add task solution

AgaMarula opened the pull request:
- [DEMO LINK](https://AgaMarula.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://AgaMarula.github.io/layout_product-cards/report/html_report/)

- [x ] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [x ] SCSS Variables are used for main values and placed in a **separate file**
- [x ] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [x ] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

BogdanMaliuta reviewed (changes requested):
Hello, your styles and markup looks good.
But look at the difference, first screen your demo, second mockup
![image](https://github.com/mate-academy/layout_product-cards/assets/156676584/ab931c32-bf48-4897-a5af-88f684443f3a)
![image](https://github.com/mate-academy/layout_product-cards/assets/156676584/a17fa813-5145-4936-8543-ccbe717f9e60)

AgaMarula commented:
Hello, 
I corrected the color of the description (Apple...)  I also changed the BEM blocks, there where supposed to be two (Card and Stars). I didn't notice it before. The test still don't pass :/  Thx for the tips.

AgaMarula commented:
Hello,
Mission completed :) Please check if everything is as it should be.

yevhenii-pyl reviewed (approved):
Mission completed 🥇
